src/inferenceql/query/parser/tree.cljc

```diff
@@ -123,6 +123,7 @@
     (walk/postwalk remove-ws node)))
 
 (defmacro match
+  "Like core.match/match, but removes whitespace nodes before matching."
   [vars & clauses]
   (let [match (macrovich/case :clj 'clojure.core.match/match
```

Comment:
I think that the cljs compiler handles this resolution on its own. If you have `clojure.blah` and that's not found, cljs will try to find `cljs.blah`. That's why `(:require [clojure.string :as ])` works in cljs, for example.

Reply:
Errr, yes?

The point I was trying to convey with the docstring is not about clj vs cljs, but that unlike plain core.match, it will filter out whitespace parse nodes before doing the matching.

Reply:
Okay! And I know I'm commenting outside of this PR...  @zane do you remember what the error was that this addresses?